x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.ts

```diff
@@ -182,7 +181,7 @@ export const buildExceptionItemEntries = ({
 }): string => {
   const and = getLanguageBooleanOperator({ language, value: 'and' });
   const exceptionItem = lists
-    .filter((t) => !entriesList.is(t))
+    .filter(({ type }) => type !== 'list')
```

Comment:
Starting to try to not rely on `io-ts` check where I can given talk of not optimal performance.